paymentsheet/src/main/java/com/stripe/android/paymentelement/confirmation/ConfirmationRegistry.kt

```diff
@@ -8,7 +8,13 @@ internal class ConfirmationRegistry(
     fun createConfirmationMediators(
         savedStateHandle: SavedStateHandle
     ): List<ConfirmationMediator<*, *, *, *>> {
-        return confirmationDefinitions.map { definition ->
+        /*
+         * We need to sort the mediators in order to guarantee the order of registered launchers. See the following
+         * documentation regarding this topic: https://developer.android.com/training/basics/intents/result#register
+         */
+        return confirmationDefinitions.sortedBy { definition ->
```

Comment:
Can you add a test to ensure this happens?

Reply:
I added tests specific to `PaymentElementConfirmationModule` but I can also add generalized unit tests for this as well.